README.md

```diff
@@ -144,6 +147,36 @@ class Node extends libp2p {
 
 `callback` is a function with the following `function (err) {}` signature, where `err` is an Error in case stopping the node fails.
 
+#### `libp2p.peerRouting.findPeer(id, callback)`
+
+> Looks up for multiaddrs of a peer in the DHT
+
+- `id`: instance of [PeerId][]
+
+#### `libp2p.contentRouting.findProviders(key, timeout, callback)`
+
+- `key`: 
+- `timeout`: Number miliseconds
+
+#### `libp2p.contentRouting.provide(key, timeout, callback)`
+
+- `key`: 
+- `timeout`: Number miliseconds
+
+#### `libp2p.dht.put(key, value, callback)`
+
+- `key`: 
+- `value`: 
```

Comment:
@dignifiedquire can you clarify here what type can be key and value? If it is "String" and "Buffer" cause the implementation lets you to, let's instead direct the users to the things we want them to actually use the DHT.